tests.py

```diff
@@ -555,8 +555,8 @@ def test_glitch_injection(self):
         max_freq_noglitch = freqs_noglitch[np.argmax(data_noglitch, axis=0)]
         max_freq_glitch = freqs_glitch[np.argmax(data_glitch, axis=0)]
         print([max_freq_vanilla, max_freq_noglitch, max_freq_glitch])
-        self.assertEqual(times_noglitch, times_vanilla)
-        self.assertEqual(times_glitch, times_vanilla)
+        self.assertTrue(np.allclose(times_noglitch, times_vanilla))
+        self.assertTrue(np.allclose(times_glitch, times_vanilla))
```

Comment:
Our times still are integers, aren't they? So shouldn't we still check for exact equality? I guess you could either cast it back to a list for `assertEqual` to digest, or there's also an `numpy.testing.assert_array_equal`.

Reply:
That's true. I'll look into `numpy.testing.assert_array_equal`

Reply:
Why not just `np.all(a == b)`, since `a` and `b` are integer arrays.

Reply:
a65eafb